tests/schema/arguments/argument_test.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+import pytest
+from tests.base.io_test import BaseIOTest
+from tests.base.value_error import BaseValueErrorTest
+
+from queenbee.schema.arguments import Arguments
+
+
+class TestIO(BaseIOTest):
+
+    klass = Arguments
```

Comment:
So this class triggers all the IO tests for files in Arguments folder? We have to document this somewhere so other developers know how the folders are set-up. Maybe add a README to tests folder.

Reply:
Done in 1d3722e32d6a3aa2c882b72bf47c9e615557fefc